### PR TITLE
minimal dist config

### PR DESCRIPTION
**Pull request by dvdn**
For minimal config understanding

### CODE REVIEW

File: config.inc.php.dist [hunk 1]
```diff
@@ -6,12 +6,12 @@ define('USER', 'guest');
 define('PASS', 'guest');
 define('VHOST', '');
 
-define('MEDIA_NAME', '');
+define('MEDIA_NAME', 'mtt');
 define('MEDIA_STORAGE_PATH', '');
```
Comment (Netmisa) [inline config.inc.php.dist:10]: I think it's better to add '/tmp/files/' here.
If you would like to add default configuration, I prefer to add for all parameters. That will be clear and not ambiguous.

What do you think ?

File: config.inc.php.dist [hunk 2]
```diff
@@ -6,12 +6,12 @@ define('USER', 'guest');
 define('PASS', 'guest');
 define('VHOST', '');
```
Comment (Netmisa) [inline config.inc.php.dist:7]: Same think (see [below](https://github.com/CanalTP/AmqpMttWorkers/pull/2/files#diff-e83d47e0e113314846de23f7425eb13cL10)). You can add 'localhost'.

File: config.inc.php.dist (hunk 1 above)
Comment (dvdn) [inline config.inc.php.dist:10]: '/tmp/...' perhaps confusing, as it needs to be exact same path as your mediamanager config path

**Comment by Netmisa** [comment]
Your pull request is realted with this one https://github.com/CanalTP/AmqpMttWorkers/pull/1

I prefer the PR #1 solution. If you are okay with it.

**Comment by dvdn** [comment]
Yes its a better way.

**Pull request closed without merging**